recipes-edge/mbed-fcce/mbed-fcce_4.13.1.bb

```diff
@@ -90,5 +90,5 @@ do_compile() {
 do_install() {
 
     install -d ${D}${EDGE_BIN}
-    install -m 755 ${S}/__Yocto_Generic_YoctoLinux_mbedtls/Debug/factory-configurator-client-example.elf ${D}${EDGE_BIN}/factory-configurator-client-armcompiled.elf
+    install -m 755 ${S}/__Yocto_Generic_YoctoLinux_mbedtls/Debug/factory-configurator-client-example.elf ${D}${EDGE_BIN}/factory-configurator-client-example.elf
```

Comment:
Even .elf is really just... Not needed.
### 
```suggestion
    install -m 755 ${S}/__Yocto_Generic_YoctoLinux_mbedtls/Debug/factory-configurator-client-example.elf ${D}${EDGE_BIN}/factory-configurator-client-example
```

Reply:
Fixed.